src/map/script.cpp

```diff
@@ -28379,7 +28393,7 @@ struct script_function buildin_func[] = {
 	BUILDIN_DEF(item_reform, "??"),
 	BUILDIN_DEF(item_enchant, "i?"),
 	BUILDIN_DEF(itemlink, "i?????????"),
-	BUILDIN_DEF(mesitemlink, "i??"),
+	BUILDIN_DEF(mesitemlink, "???"),
```

Comment:
```suggestion
	BUILDIN_DEF(mesitemlink, "v??"),
```
`'v' - value (either string or int32 or reference)`